cilium/cmd/debuginfo.go

```diff
@@ -41,13 +45,26 @@ const (
 	STDOUT outputType = 0 + iota
 	MARKDOWN
 	HTML
+	JSONOUTPUT
+	JSONPATH
+)
+
+var (
+	// Can't tall it jsonOutput because another var in this package uses that.
+	jsonOutputDebuginfo = "JSON"
+	markdownOutput      = "MARKDOWN"
+	htmlOutput          = "HTML"
+	jsonpathOutput      = "JSONPATH"
+	re                  = regexp.MustCompile(`^jsonpath\=(.*)`)
```

Comment:
This would be easier to understand its usage in the later code if it were named something like `jsonPathRegex`.